scripts/mail-alarm

```diff
@@ -676,60 +661,68 @@ class XapiMessage:
 
 def main():
     if len(sys.argv) < 2:
-        log_err("Expected at least 1 argument but got none: [\"%s\"]." % (' '.join(sys.argv)))
-        raise Exception("Insufficient arguments")
+            log_err("Expected at least 1 argument but got none: [\"%s\"]." % (' '.join(sys.argv)))
+            raise Exception("Insufficient arguments")
+    
+    
+    try:
+        session = XenAPI.xapi_local()
+        ma_username="__dom0__mail_alarm"
+        session.xenapi.login_with_password(ma_username, "", "1.0", "xen-api-scripts-mail-alarm")
```

Comment:
I think this makes sense to be out of the `try:` block: it doesn't make sense to logout when we're not logged in yet.